src/Components/App/App.js

```diff
@@ -0,0 +1,95 @@
+import React, {
+  useState,
+  useEffect,
+  useRef,
+  useMemo,
+  useCallback,
+} from 'react';
+import * as Tone from 'tone';
+import './App.css';
+import createSynth from '../SynthEngine/SynthEngine';
+import keyboardSwitch from '../../util/keyboardSwitch';
+
+const { oscillators, delay, reverb, filter } = createSynth();
+const engine = oscillators.chain(delay, reverb, filter, Tone.Destination);
```

Comment:
I still wonder if there's a way to not have these (lines 13 and 14) globally set, but we can circle back to that.